assets/src/components/v2/dup/page_load_no_data.tsx

```diff
@@ -0,0 +1,39 @@
+import useOutfrontStation from "Hooks/use_outfront_station";
+import React, { ComponentType } from "react";
+import LinkArrow from "../bundled_svg/link_arrow";
+import Loading from "../bundled_svg/loading";
+import NormalHeader from "./normal_header";
+
+// Fix station name tags without rider-facing names
+const REPLACEMENTS = {
+  WTC: "World Trade Center",
+  Malden: "Malden Center",
+} as {[key:string]: string};
```

Comment:
Not sure the best place for it, but would it be worth pulling this bit into something common so this and `no_data` can use the same list? Or you could just leave it in this and export it to the other (or vice versa).